digitalearthau/config/eo3/metadata_eo3.yaml

```diff
@@ -0,0 +1,85 @@
+---
+name: eo3
+description: Default EO3 with no custom fields
+dataset:
+    id: [id]  # needs to be fixed
```

Comment:
I know I put this comment in myself `# needs to be fixed` but it can be misinterpreted as "needs fixing" while it really means "has to be this value and not any other"

Reply:
Updated the comment